src/data/ACTIONS/BLM.js

```diff
@@ -123,9 +143,9 @@ export default {
 		icon: 'https://xivapi.com/i/000000/000466.png',
 		cooldown: 8,
 	},
-	CONVERT: {
+	MANAFONT: { // For now, I'm going to assume SE just re-named the action. Updatable if wrong.
```

Comment:
Keep the comment until confirmed with final data, but looks like convert's ID hasn't changed in the update. Should be safe.

Reply:
Yeah, I tried to leave comments where I wasn't completely sure about things. I'll definitely comb back through the definitions once we know IDs and stuff to adjust it if it's wrong.